tasks/InsightAppSec/helpers/insightAppSecApi.ts

```diff
@@ -293,7 +293,7 @@ export default class InsightAppSecApi
                 xhr.setRequestHeader("X-Api-Key", this.apiKey);
                 xhr.setRequestHeader("Content-Type", "application/json");
                 xhr.setRequestHeader("Accept", "application/json");
-                xhr.setRequestHeader("User-Agent", "r7:insightappsec-azure-devops-extension/1.0.5");
+                xhr.setRequestHeader("User-Agent", "r7:insightappsec-azure-devops-extension/1.0.6");
```

Comment:
xmlhttprequest adds some default headers, one of them being User-Agent.

When setting a header that already exists in the headers list xmlhttprequest will append the new value to any existing values:
headers[header] = headers[header] ? headers[header] + ', ' + value : value;
 
There appears to be no reliable way to remove the default value set by xmlhttprequest. The best solution may be to use another request library such as axois

Reply:
Yeah, I was afraid of that 😢 . I'm a big fan of axios but what are your overall thoughts? We could leave it for now as we focus on more important stuff and come back to it down the road

Reply:
Exactly what I was thinking, leave it for now, but keep it in mind when I have some time. I have also added the same details to the JIRA task specific to this item.